scarlet/component.py

```diff
@@ -263,7 +263,16 @@ def B(self):
         """Number of bands
         """
         return self.components[0].B
-
+    @property
```

Comment:
Why are those needed? You should not need to talk to the components directly as the resampling only operates on the full scene

Reply:
Old stuff that I thought I neede. 
Deleted